app/views/automobiles/show.html.haml

```diff
@@ -0,0 +1,16 @@
+.flex.justify-center.mt-8.mr-8.ml-8
+    .max-w-lg.rounded.overflow-hidden.shadow-lg{id: dom_id(@automobile)}
+        %img.w-full{src: "https://www.christenseninstitute.org/wp-content/uploads/2016/08/nonconsumption-ford-model-t-1.jpg"}/
+        .px-6.py-4
+            .font-bold.text-xl.mb-2= "#{@automobile.brand} #{@automobile.model}"
+            %p.text-grey-darker.text-base= "#{@automobile.description}"
+        .px-6.py-4
+            %span.inline-block.bg-grey-lighter.rounded-full.px-3.py-1.text-sm.font-semibold.text-grey-darker.mr-2="Year: #{@automobile.year}"
+            %span.inline-block.bg-grey-lighter.rounded-full.px-3.py-1.text-sm.font-semibold.text-grey-darker="Price: #{@automobile.price} SEK per day"
+            %span.inline-block.bg-grey-lighter.rounded-full.px-3.py-1.text-sm.font-semibold.text-grey-darker
```

Comment:
That is an empty span :( shouldn't here be the description?

Reply:
Yeah... wasn't done with it yet.  I just pushed up a clean copy so Adi could pull.